### PR TITLE
Save recipe backend

### PR DESCRIPTION
**Pull request by ben-dh3**
- create recipe function working, tests passing
- update recipe function built, not tested

### CODE REVIEW

**Review by kawrou — APPROVED** [review]
Looks good. I can copy some of that for getRecipes 👍

File: api/routes/recipes.js [hunk 1]
```diff
@@ -5,4 +5,9 @@ const router = express.Router();
 
 router.get("/scrape-recipe", RecipesController.fetchRecipeData);
 
+router.post("/", RecipesController.create);
+
+// patch request most suitable - https://www.geeksforgeeks.org/what-is-the-difference-between-put-post-and-patch-in-restful-api/
```
Comment (kawrou) [inline api/routes/recipes.js:10]: Good use of comment in code 🥇

**Review by kawrou — APPROVED** [review]
Thanks for the link

**Review by Ro-0171 — APPROVED** [review]
Looks great!

**Pull request merged**